sdk/nodejs/applicationloadbalancing/ipAddressType.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export let Ipv4:      IpAddressType = "ipv4";
```

Comment:
Should these be `IPv4` and `DualStack`?  I'm not really sure what our guidelines are here, but they seem to be read a little weird as is.

Reply:
we don't generally do this.  we would just have the `export type` below and leave it at that.  users would then get (in tools like VSCode) autocompleting to `"ipv4"` and `"dualstack"`

Reply:
pulumi/aws already has a type called `export class IPSet`.  So let's capitalize this to ` export type IPAddressType =`

Thanks!

Reply:
changed these to `export const` after convo in Slack